src/house.cpp

```diff
@@ -419,7 +418,12 @@ void AccessList::parseList(const std::string& list)
 	std::istringstream listStream(list);
 	std::string line;
 
+	int lineNo = 1;
 	while (getline(listStream, line)) {
+		if (++lineNo > 100) {
```

Comment:
I think we should approach this with not more than 50, I'd honestly doubt that someone is really willing to put 100 different characters in there for a real reason.
I would also change ```line.length() > 100``` to something lower aswell, as we only support one character per line now and I honestly doubt anyone is having characters in his database which are probably longer than 25 letters in length.

Reply:
Rather than finding the lowest limit we can think would work, my goal is to establish an upper bound that is extremely unlikely to be problematic for normal usage.
Unless there is still concern for abuse with current limits, I think 100x100 is fine.

Reply:
Why don't you just make it configurable through a config variable that goes along with the other house settings? In my eyes it wouldn't be much different from making max packets per second configurable.